subgraphs/cbridge/README.md

```diff
@@ -0,0 +1,84 @@
+# cBridge 
+
+cBridge introduces the best-in-class cross-chain token bridging experience with deep liquidity for users, highly efficient and easy-to-use liquidity management for both cBridge node operators and Liquidity Providers who do not want to operate cBridge nodes, and new exciting developer-oriented features such as general message bridging for cases like cross-chain DEX and NFTs. 
+
+## Usage Metrics
+
+`Active Users`, `Total Unique Users` & `Daily Transaction Count`
+
+Transactions of interest include:
+Cross-chain transfer of Tokens
+Cross-chain transfer of NFTs
```

Comment:
The schema currently doesn’t support NFT transfers out of the box. But we can still count them as transactions / messages. And keep track of the users using that feature. Let’s merge this as it is, and if we discard this information I’ll amend it

Reply:
sounds good